sktime/transformations/series/summarize.py

```diff
@@ -32,14 +32,31 @@ class WindowSummarizer(BaseTransformer):
     lag_feature: dict of str and list, optional (default = dict containing first lag)
         Dictionary specifying as key the type of function to be used and as value
         the argument `window`.
-        For all keys other than `lag`, the argument `window` is a length 2 list
-        containing the integer `lag`, which specifies how far back
-        in the past the window will start, and the integer `window length`,
-        which will give the length of the window across which to apply the function.
-        For ease of notation, for the key "lag", only a single integer
+
+        For all keys other than `lag`, the argument `window` is a length 2 list.
+        The window elements can be either integers indicate the number of
+        observations, or strings specifying a time period (an offset).
+
+        The first element of the `window` is the `lag`, which specifies how far back
+        in the past the window will start. The second element, `window length`,
+        is the length of the window across which to apply the function.
+
+        For ease of notation, for the key "lag", only a single integer or offset
         specifying the `lag` argument will be provided.
 
-        Please see blow a graphical representation of the logic using the following
+        The offsets work only with datasets with time index (either pd.DateimeIndex,
```

Comment:
the above text is better! But this section is unclear. I suggest to make it much clearer that you are talking about a sub-case.

E.g., "For offsets of string format..."